go.mod

```diff
@@ -69,3 +69,5 @@ require (
 	k8s.io/mount-utils v0.27.1 // indirect
 	k8s.io/utils v0.0.0-20230406110748-d93618cff8a2 // indirect
 )
+
+replace github.com/longhorn/backupstore v0.0.0-20230830075002-fa25b1a97ffd => github.com/derekbit/backupstore v0.0.0-20230906134210-074ab4a06389
```

Comment:
The https://github.com/longhorn/backupstore/pull/145 is merged. I think we can just update `longhorn/backupstore` version?